src/components/webview/MainWebviewProvider.ts

```diff
@@ -664,4 +707,91 @@ export class IntuitaProvider implements WebviewViewProvider {
 		}
 		return mappedNode;
 	};
+
+	private __buildCaseByFolderTree = (
+		element: FileElement,
+		parentFolderPath: string,
+	): TreeNode | null => {
+		if (!element.children || element.children.length === 0) {
```

Comment:
now I am thinking we should ensure `element` children are always just an non-nullable array (let's do this change in another PR)